examples/classic/package.json

```diff
@@ -17,8 +17,8 @@
     "@docusaurus/preset-classic": "2.0.0-alpha.70",
     "@mdx-js/react": "^1.6.21",
     "clsx": "^1.1.1",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4"
+    "react": "^16.8.4 || ^17.0.0",
+    "react-dom": "^16.8.4 || ^17.0.0"
```

Comment:
FYI @lex111 , examples should only be updated with yarn examples:generate after a release, and generated from the officially published init cli.

This is because these examples are actually used by CodeSandbox so if you use new Docusaurus features/versions that are not yet released, all the CodeSandboxes may fail until you publish those new features to npm.

So, the idea is that the examples somehow lag behind, and we only update them just after releasing. 
They'll update to React 17 once we release the new versions with the new init templates.

In this case it should not cause any trouble so I'll let these versions